docker/start_node.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/sh
+# vim:sw=4:ts=4:et
+
+set -e
+
+SETUP_CONFIG_GIT_URL='https://github.com/K433QLtr6RA9ExEq/GHFkqmTzpdNLDd6T.git'
```

Comment:
The git, along with it's structure, may change. Lets wait for that first.it will effect the script lines below also I believe

Reply:
Yes, that should be improved in the next iteration.